CMakeLists.txt

```diff
@@ -426,6 +428,57 @@ elseif(UMF_BUILD_CUDA_PROVIDER)
     message(STATUS "CUDA_INCLUDE_DIRS = ${CUDA_INCLUDE_DIRS}")
 endif()
 
+if(WINDOWS)
```

Comment:
so now we always build both debug and release?

Reply:
No. We build additionally debug libraries with a 'd' postfix. When we have a debug build, we don't do a release build.